cms/envs/common.py

```diff
@@ -966,6 +966,7 @@
     # Standard apps
     'django.contrib.auth',
     'django.contrib.contenttypes',
+    'django.contrib.humanize',
```

Comment:
What class is used from `humanize`? Just curious.

Reply:
This is required by the new apps included below, all of which are necessary to include `wiki`.